src/.vuepress/components/ForkButtons.vue

```diff
@@ -6,9 +6,11 @@
 		<button class="githubForkButton" :style="githubForkStyle" :onclick="githubForkLink">
 			{{ githubForkLabel }}
 		</button>
-		<br>
-		<span class="versionNotice">Requires <strong>Android {{androidVersion}}</strong> or higher.</span>
-	</div>	
+		<br />
+		<span class="versionNotice"
+			>Requires <strong>Android {{ androidVersion }}</strong> or higher.</span
+		>
```

Comment:
This looks weird. I guess that the linter got confused?
```suggestion
		<span class="versionNotice">Requires <strong>Android {{ androidVersion }}</strong> or higher.</span>
```

Reply:
The troublesome part with the linter doing this is that any change will just revert it back unless you eslint disable the chunk it seems like

Reply:
Could use `"htmlWhitespaceSensitivity": "ignore"` in `.prettierrc`. Which fixes this partially, it would look like this
```html
<span class="versionNotice">
  Requires
  <strong>Android {{ androidVersion }}</strong>
  or higher.
</span>
```
instead of this
```html
<span class="versionNotice"
  >Requires <strong>Android {{ androidVersion }}</strong> or higher.</span
>
```

Reply:
Sure, that looks better